ha/alert_monitor/node_alert_monitor.py

```diff
@@ -0,0 +1,35 @@
+'''Dervied module in the Monitoring hierarchy which monitors node alert'''
+
+class NodeAlertMonitor():
+    '''
+    Module responsible for analyzing an alert information
+    to generate an IEM and sending the IEM across.
+    '''
+    #TODO: Inherit this class from base AlertMonitor
+
+    # TODO: This can be initialized at base level
```

Comment:
I think lets move these to some common file (e.g. const.py)

Reply:
Sure. Done.